package.json

```diff
@@ -25,18 +25,22 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "vscode-chrome-debug-core": "git+https://github.com/Microsoft/vscode-chrome-debug-core.git#v2",
+    "reflect-metadata": "^0.1.13",
     "vscode-debugadapter": "^1.32.1",
     "vscode-nls": "^4.0.0"
   },
   "devDependencies": {
+    "@types/http-server": "^0.10.0",
     "@types/lodash": "^4.14.120",
     "@types/mocha": "^2.2.48",
     "@types/chai": "^4.1.7",
     "@types/chai-string": "^1.4.1",
     "@types/mockery": "^1.4.29",
     "@types/node": "^8.10.39",
     "@types/puppeteer": "^1.12.3",
-    "@types/source-map": "^0.1.27",
+    "@types/request": "^2.48.1",
+    "@types/request-promise-native": "^1.0.15",
+    "@types/semver": "^6.0.0",
```

Comment:
What are these semver types for?

Reply:
Request with promisify wasn't typing properly in strict mode, so I changed it to request-promise-native.
I don't think we need semver or request any more so I'll remove those.